sunpy/io/special/asdf/tags/tests/test_coordinate_frames.py

```diff
@@ -63,10 +63,7 @@ def test_hgc_100():
 
 # Skip these two tests on windows due to a weird interaction with atomicfile
 # and tmpdir
-skip_windows_asdf = pytest.mark.skipif(
-    (LooseVersion(asdf.__version__) < LooseVersion("2.3.1")
-     and platform.system() == 'Windows'),
-    reason="See https://github.com/spacetelescope/asdf/pull/632")
+skip_windows_asdf = pytest.mark.skipif(platform.system() == 'Windows', reason="See https://github.com/spacetelescope/asdf/pull/632")
```

Comment:
Do we still need to skip this?

Reply:
Oh yeah this PR seems to have been merged, I don't think we need a skip